third_party/dependency_analyzer/src/main/io/bazel/rulesscala/dependencyanalyzer/compiler/BUILD

```diff
@@ -5,7 +5,9 @@ load("@io_bazel_rules_scala//scala:scala_cross_version.bzl", "version_suffix")
 scala_library_for_plugin_bootstrapping(
     name = "dep_reporting_compiler",
     srcs = select({
```

Comment:
Since you are hiding/unifying version specific repositories under `@scala_compiler_sources` maybe this `select` should be moved into that repository. I mean `@scala_compiler_sources` would have a single alias target with select to version specific sources. For example:
```
alias(
  name = "src",
  visibility = ["//visibility:public"],
  actual = select({"@io_bazel_rules_scala_config//:scala_version_2_12_19": "@scala_compiler_source_2_12_19//:src"}),
)
```
And here in compilation target you would specify `srcs = ["@scala_compiler_sources//:src"],`

What do you think?

Also I think that compilation target could go into that new repository, but let's not go down that rabbit hole now.

Reply:
I like it. Done.